ci-operator/step-registry/gcp/provision/static-creds/gcp-provision-static-creds-ref.yaml

```diff
@@ -0,0 +1,25 @@
+ref:
+  as: gcp-provision-static-creds
```

Comment:
I think we can use the same name convention as AWS `aws-provision-cco-manual-users-static`.

Reply:
agree.